echopype/tests/echodata/test_echodata_combine.py

```diff
@@ -211,13 +212,76 @@ def test_combine_echodata(raw_datasets):
                 del test_ds.attrs["conversion_time"]
                 del combined_group.attrs["conversion_time"]
 
-        if (combined_group is not None) and (test_ds is not None):
-            assert test_ds.identical(combined_group.drop_dims(grp_drop_dims))
+        if group_name != "Provenance":
+            # TODO: Skip for Provenance group for now, need to figure out how to test this properly
+            if (combined_group is not None) and (test_ds is not None):
+                assert test_ds.identical(combined_group.drop_dims(grp_drop_dims))
+
+
+@pytest.mark.parametrize("test_param", [
+        "single",
+        "multi"
+    ]
+)
+def test_combine_echodata_combined_and_others(ek60_multi_test_data, test_param, sonar_model="EK60"):
+        """
+        Integration test for combine_echodata with
+        a single combined ed and a single echodata
```

Comment:
```suggestion
        Integration test for combine_echodata with the following cases:
        - a single combined echodata object and a single echodata object
        - a single combined echodata object and 2 single echodata objects
```
I also wonder if there should be a test for:
- a single combined echodata object and another combined echodata object

something like:
```
combined_ed = echopype.combine_echodata(eds[:2])
combined_ed_other = echopype.combine_echodata(eds[2:])
combined_ed2 = echopype.combine_echodata([combined_ed, combined_ed_other])
```